src/10_functions.py

```diff
@@ -1,6 +1,13 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
 # YOUR CODE HERE
+def is_even(n):
+    if n % 2 == 0:
+        print('Even!')
+        return True
+    else:
+        print('not even :/')
+        return False
```

Comment:
Nice!! 

Everything here is perfect, but the return statements aren't required unless you plan on using the return value. 

A bit of a pedantic critique, but just wanted to make sure you didn't think the function required them for any reason.